### DISCUSSION TITLE
Unknown column after moving from ver. 4.6.7 to ver. 6.0.1

### DISCUSSION THREAD

**Discussion by MsideSyncro** [snipe/snipe-it#11183] [Q&A] [answered]
Hello all, I am having troubles with a new install of Snipe-IT on an Ubuntu server. I am 100% new to this and Data bases in general. I was tasked with setting it up and now migrating from the current running on a Windows Server 2012 machine.

Current:
Snipe IT Version: 4.6.7
MySQL MariaDB
Windows Server 2012 R2

Moving to:
Snipe-IT Version 6.0.1
MySQL MariaDB
Ubuntu Server 22.04

I am encountering an error when logging in after the setup and migration completed.
Below is from the debug

Illuminate\Database\QueryException
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'accessories_users.note' in 'field list' (SQL: select `accessories`.*, `accessories_users`.`assigned_to` as `pivot_assigned_to`, `accessories_users`.`accessory_id` as `pivot_accessory_id`, `accessories_users`.`id` as `pivot_id`, `accessories_users`.`created_at` as `pivot_created_at`, `accessories_users`.`note` as `pivot_note` from `accessories` inner join `accessories_users` on `accessories`.`id` = `accessories_users`.`accessory_id` where `accessories_users`.`assigned_to` in (376))

I have looked through the most common issues and tried the list of php commands outlined 

composer dump-autoload
php artisan cache:clear 
php artisan config:clear
php artisan route:clear
php artisan debugbar:clear
php artisan event:clear
php artisan view:clear
php artisan optimize:clear
php artisan clear-compiled

No luck

Tried re-migrating and clearing config

Knowing nothing about SQL, I feel I am in over my head on how to work this out.
Anyone else run into this or can point me in the right direction?
I can try and provide any other info that may be needed.

Thank you,

**Comment by MsideSyncro** [comment, marked as answer]
I was able to resolve this on my own. There was a permissions issue with that specific table and adding the following resolved that
GRANT ALL PRIVILEGES ON name.* TO 'name'@'localhost';
FLUSH PRIVILEGES;
Had a couple more migration issue that I was able to resolve easily

**Comment by uberbrady** [comment]
AWESOME! Thanks for letting us know!
